osidb/models.py

```diff
@@ -1484,6 +1484,19 @@ def _validate_notaffected_open_tracker(self):
                 "NOTAFFECTED but has open tracker(s).",
             )
 
+    def _validate_ooss_open_tracker(self):
+        """
+        Check whether out of support scope products have open trackers.
+        """
+        if (
+            self.resolution == Affect.AffectResolution.OOSS
+            and self.trackers.exclude(status__iexact="CLOSED").exists()
```

Comment:
could you document why you're using `iexact` here? And are we sure `CLOSED` covers all? especially for JIRA trackers